src/main/converter/opds.ts

```diff
@@ -57,7 +57,7 @@ export class OpdsFeedViewConverter {
         const title = convertMultiLangStringToString(metadata.Title);
         const authors = convertContributorArrayToStringArray(metadata.Author);
         const publishers = convertContributorArrayToStringArray(metadata.Publisher);
-        const tags = metadata.Subject.map((subject) => subject.Name);
+        const tags = metadata.Subject.map((subject) => subject.Name as string);
```

Comment:
This forced type coercion to `string` is incorrect. The correct implementation is in this older Pull Request:
https://github.com/readium/readium-desktop/pull/202

Reply:
Fixed in commit: https://github.com/readium/readium-desktop/pull/212/commits/5ec111bf7f326759c1e39ac643da82c4786943a7